ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/zoomToHome.tsx

```diff
@@ -25,16 +25,16 @@ type Props = {
 }
 const Button = styled.button`
   flex: 1;
+  margin-left: ${props => props.theme.mediumSpacing};
+  margin-right: ${props => props.theme.mediumSpacing};
```

Comment:
and here:
```suggestion
  margin-right: 0 ${props => props.theme.mediumSpacing} 0 ${props => props.theme.mediumSpacing};
```

Reply:
IMO the explicit attributes are more readable if only two sides are being set